api-outbound.yaml

```diff
@@ -120,6 +120,57 @@ paths:
         basicauth:
           username: "${NEXPOSE_API_USERNAME}"
           password: "${NEXPOSE_API_PASSWORD}"
+  /api/3/:
```

Comment:
Question: this trailing slash isn't used AFAIK in the actual check. Will the proxy allow that, or should we just remove this trailing slash?

Reply:
i think it should be fine